drops/aio_quickstart_bash_scripts.json

```diff
@@ -0,0 +1,24 @@
+{
+  "Title": "Azure IoT Operations (AIO) Quickstart Bash Scripts",
```

Comment:
I would change it to be more descriptive into what is doing -> Azure IoT Operations (AIO) using GitHub Codespaces